change-validation/src/main/scala/com/normation/plugins/changevalidation/NotificationService.scala

```diff
@@ -128,13 +129,17 @@ class NotificationService(
   def sendNotification(step: WorkflowNode, cr: ChangeRequest): IOResult[Unit] = {
     for {
       serverConfig  <- getSMTPConf(configMailPath)
-      emailConf     <- getStepMailConf(step, configMailPath)
-      rudderBaseUrl <- getRudderBaseUrl(configMailPath)
-      params        =  extractChangeRequestInfo(rudderBaseUrl, cr)
-      mf            =  new DefaultMustacheFactory()
-      emailBody     <- getContentFromTemplate(mf, emailConf, params)
-      emailSubject  <- getSubjectFromTemplate(mf, emailConf.subject, params)
-      _             <- emailService.sendEmail(serverConfig, emailConf.toEnvelop(emailBody).copy(subject = emailSubject))
+      _             <- ZIO.when(serverConfig.smtpHostServer.nonEmpty) {
+                         for {
```

Comment:
I think there should be an info log line in each case - oh, no the config is read on each notification. 
OK, we need to add the info in the plugin page in a next version. (ie in the plugin page, something like "Email notification are [disabled (configure email server in /... to enable) | enable (comment config in .... to disable) ]

Reply:
Do you mean we want to log whether active or not when we try to send an email ?

Reply:
yes, but I forgot that we reread conf each time, so it would hard to spot when it changes from enable to disable (or opposite). 
So in place of having a log when the config is read, we need to display the info somewhere else.